src/shared/components/ncTable/partials/columnTypePartials/forms/TextLineForm.vue

```diff
@@ -42,23 +42,32 @@
 import { translate as t } from '@nextcloud/l10n'
 
 export default {
-	name: 'TextLineForm',
+
 	props: {
 		column: {
 			type: Object,
 			default: null,
 		},
 	},
+
 	data() {
 		return {
 			mutableColumn: this.column,
 		}
 	},
+
+	computed: {
+		getMaxLength() {
+			return this.mutableColumn.textMaxLength >= 0 ? this.mutableColumn.textMaxLength : null
```

Comment:
A max text length of 0 doesn't make sense, right? Should it then just also be unlimited?